ui/apps/platform/cypress/integration/vulnerabilities/VulnerabilityReporting/reportForm.test.ts

```diff
@@ -52,7 +52,7 @@ describe('Vulnerability Reporting form', () => {
         cy.get('button:contains("Save")').click();
     }
 
-    function validateReportParameters(params: [string, string][]) {
+    function validateReportParameters(params: string[][]) {
```

Comment:
For my learning, can you explain this change?

Given function calls in the companion contribution, I would have written the type as original  at the left.

Reply:
Oh, yeah. Without type declaration or maybe `as const` for `expectedReportParameters` TypeScript assumes that the code could have pushed items that do not satisfy the stricter type?